webapp/channels/src/components/activity_and_insights/insights/insights.tsx

```diff
@@ -46,9 +46,7 @@ const Insights = () => {
     let playbooksEnabled = false;
     if (products) {
         products.forEach((product) => {
-            if (product.pluginId === suitePluginIds.boards) {
```

Comment:
This seems like there might have been a bug here before since this previously didn't work with the Boards plugin. I think we likely want to keep this line but have it check for `suitePluginIds.focalboard`

Reply:
Do we know for a fact that this should/will work with the plugin? The feature is planned to be removed and [is effectively disabled right now](https://mattermost.atlassian.net/browse/MM-53026), so I'd prefer to add plugin support if we're sure it's a requirement

Reply:
I'm not sure. The frontend API for product/plugin never had a chance to diverge, so I'd assume that it would still work, but I don't know at all. I'm fine to leave this as-is since we can always re-add it if anything changes